ready_recipes/arm_pyart/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.6.0" %}
+
+package:
+    name: arm_pyart
+    version: {{ version }}
+
+source:
+    fn: arm_pyart-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/a/arm_pyart/arm_pyart-{{ version }}.tar.gz
+    md5: 6c382dda78193c730a6a47d2d7b18eeb
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python
+        - numpy x.x
+        - trmm_rsl   # [not win]
+
+    run:
+        - python
+        - numpy
```

Comment:
Looks like there is not a version of NumPy 1.9 version of netcdf4 for Python 3.5.  Let me see if I can skip that build.